doc/Scorer.rst

```diff
@@ -1,64 +1,83 @@
 .. _scorer-scripts:
 
-External scorer scripts
-=======================
+How to Train a Language Model
+=============================
 
-🐸STT pre-trained models include an external scorer. This document explains how to reproduce our external scorer, as well as adapt the scripts to create your own.
+Introduction
+------------
 
-The scorer is composed of two sub-components, a KenLM language model and a trie data structure containing all words in the vocabulary. In order to create the scorer package, first we must create a KenLM language model (using ``data/lm/generate_lm.py``, and then use ``generate_scorer_package`` to create the final package file including the trie data structure.
+This document explains how to train and format a language model for deployment.
 
-The ``generate_scorer_package`` binary is part of the native client package that is included with official releases. You can find the appropriate archive for your platform in the `GitHub release downloads <https://github.com/coqui-ai/STT/releases/latest>`_. The native client package is named ``native_client.{arch}.{config}.{plat}.tar.xz``, where ``{arch}`` is the architecture the binary was built for, for example ``amd64`` or ``arm64``, ``config`` is the build configuration, which for building decoder packages does not matter, and ``{plat}`` is the platform the binary was built-for, for example ``linux`` or ``osx``. If you wanted to run the ``generate_scorer_package`` binary on a Linux desktop, you would download ``native_client.amd64.cpu.linux.tar.xz``.
+In practice you will almost always want to deploy a language model. A good language model will improve transcription accuracy by correcting predictable spelling and grammatical mistakes. If you can predict what kind of speech your 🐸STT will encounter, you can make great gains in accuracy with a custom language model.
```

Comment:
nit: Starting with "in practice" implies some previously discussed alternative. Feels weird to me. Plus the "almost". Like the alternative exists but it's not discussed anywhere.

Reply:
I discuss this later down, but I see your point, esp. about "in practice"